internals/cli/ui/io.go

```diff
@@ -20,6 +20,8 @@ var (
 type IO interface {
 	Input() io.Reader
 	Output() io.Writer
+	Stdin() *os.File
+	Stdout() *os.File
```

Comment:
I don't see these being used. (where) are they needed?

Reply:
They are meant for the pagination functionality. But technically `run.go` also needs `Stdout`. So introduced them there.

Reply:
So this is the main fix. We're exposing `*os.File` through a separate function that can be called when a function needs direct access to `Stdin` or `Stdout`.

Reply:
I'm not sure whether I got this right, but is it about the underlying implementation being checked whether it is `os.Stdout`? If so, couldn't we also return it as an `io.Writer`, as long as the implementation is `os.Stdout` itself and not a wrapped version?

Reply:
That would work, yes. But that would not garantuee that it actually is an `*os.File`.